code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -232,12 +232,18 @@ s
 		stop_automated_movement = FALSE
 		SSmove_manager.stop_looping(src)
 
-// Tarantulas are the balanced generalist of the spider family: Moderate stats all around.
-/mob/living/simple_animal/hostile/poison/giant_spider/tarantula
-	name = "tarantula"
+// Net casters are the balanced generalist of the spider family: Moderate stats all around, and a ranged knockdown to assist others
+/mob/living/simple_animal/hostile/poison/giant_spider/netcaster
+	name = "net caster"
 	obj_damage = 35
 	speed = 0.5
 	onweb_speed = 0
+	var/obj/effect/proc_holder/spider/throw_web/spidernet
```

Comment:
add RemoveAbility() to Destroy()

Reply:
👍

I'd like to point out this isn't done for any other `effect/proc_holder` anywhere in code except for broodmothers and wrap. Should this be happening with all the rest of them too?

Reply:
This has been done, leaving open for you to resolve it

Reply:
Dunno about the rest of the code, but any objects stored should be qdeled when the parent Destroys

Reply:
I went ahead and applied it to the ability that had been previously overlooked as well. It's documented above now. 

![image](https://user-images.githubusercontent.com/9547572/231897787-f5b73848-369c-4910-9bc1-91efe1b9a700.png)